lib/modules/showImages.js

```diff
@@ -1268,7 +1266,48 @@ function generateIframe(options) {
 	element.independent = true;
 
 	keepMediaVisible(iframeNode);
-	makeMediaIndependentOnResize(element, iframeNode);
+	makeMediaIndependentOnResize(element, element);
+
+	// Handle resize for iframe
+	let initialWidth, initialDiagonal, minWidth, left, top, ratio;
+
+	function getDiagonal(x, y) {
+		const w = Math.max(1, x - left);
+		const h = Math.max(1, y - top);
+		return Math.round(Math.hypot(w, h));
+	}
+
+	addDragListener({
+		media: dragHandle,
+		atShiftKey: false,
+		onStart(x, y) {
+			// stop iframe stealing events
+			resizeHelper.style.pointerEvents = 'all';
+
+			({ left, top } = iframeNode.getBoundingClientRect());
+
+			initialDiagonal = getDiagonal(x, y);
+			initialWidth = iframeNode.clientWidth;
+			// Calc ratio
+			ratio = iframeNode.clientHeight / iframeNode.clientWidth;
+
+			minWidth = Math.max(1, Math.min(iframeNode.clientWidth, 100));
+		},
+		onMove(x, y) {
+			const newWidth = Math.max(minWidth, getDiagonal(x, y) / initialDiagonal * initialWidth);
+			const newHeight = `${newWidth * ratio}px`;
+
+			resizeMedia(iframeNode, newWidth);
+			iframeNode.style.height = newHeight;
```

Comment:
To avoid setting width / height on all these elements, rather use a wrapper which you update the dimensions for, and set the height / width of the children to `100%`.